### PR TITLE
CORTX-30117: Adding two tests for cluster shutdown

### PR DESCRIPTION
**Pull request by zohebkhann**
Signed-off-by: Zoheb Khan <zoheb.khan@seagate.com>

# Problem Statement

- Added two tests for the cluster shutdown feature
- Modified the ha lib to exclude m0client service health check status

Design
# Design
- https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/706938164/F-8B-F-8C+Cluster+shutdown+-+Immediate

# Coding
   Checklist for Author
-  [*] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [*] New/Affected tests are executed on Latest Build
-  [*] Attach test execution logs
-  [*] Collection tested and no collection error introduced (`pytest --local True --collect-only`)
- http://cortx-storage.colo.seagate.com/logs1/ldr-r2/zk/CORTX-30117/

# Review Checklist 
  Checklist for Author
- [*] JIRA number/GitHub Issue added to PR
- [*] PR is self reviewed
- [*] Jira and state/status is updated and JIRA is updated with PR link
- [*] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: libs/ha/ha_common_libs_k8s.py [hunk 1]
```diff
@@ -781,9 +781,10 @@ def check_cluster_status(self, pod_obj, pod_list=None):
                 command_suffix=f"-c {common_const.HAX_CONTAINER_NAME} -- "
                                f"{common_cmd.MOTR_STATUS_CMD}", decode=True)
             for line in res.split("\n"):
-                if "failed" in line or "offline" in line or "unknown" in line:
-                    LOGGER.error("Response for data pod %s's hctl status: %s", pod_name, res)
-                    return False, f"Cortx HCTL status has Failures in pod {pod_name}"
+                if 'm0_client' not in line:
```
Comment (gaurkrgaurav) [inline libs/ha/ha_common_libs_k8s.py:784]: Please try to use assert

Comment (zohebkhann) [reply libs/ha/ha_common_libs_k8s.py:784]: Assert is called in the caller function, which takes decision based on the returned value from this function.

File: libs/motr/motr_core_k8s_lib.py [hunk 2]
```diff
@@ -348,6 +350,27 @@ def md5sum_cmd(self, file1, file2, node):
         assert_utils.assert_not_in("ERROR" or "Error", resp,
                                    f'"{cmd}" Failed, Please check the log')
 
+    def get_md5sum(self, file, node):
```
Comment (seagate-sarang-sawant) [inline libs/motr/motr_core_k8s_lib.py:353]: This function is as good as issuing a command into a Cortx container. If we anyway want to have that , can you add a suffix (just for ref hax_container suffix) to function name ?

**Review by seagate-sarang-sawant — APPROVED** [review]

**Review by gaurkrgaurav — APPROVED** [review]

File: libs/motr/motr_core_k8s_lib.py (hunk 2 above)
Comment (zohebkhann) [reply libs/motr/motr_core_k8s_lib.py:353]: All motr utilities run from hax container, so I have added the description to this function mentioning the same. The same applies to dd, m0cat, m0cp and diff functions.

**Review by seagate-sarang-sawant — APPROVED** [review]

**Pull request merged**